src/LatteTemplateResolver/NetteApplicationUIControl.php

```diff
@@ -4,41 +4,22 @@
 
 namespace Efabrica\PHPStanLatte\LatteTemplateResolver;
 
-use Efabrica\PHPStanLatte\Template\Template;
-use Efabrica\PHPStanLatte\Template\Variable;
-use PHPStan\BetterReflection\Reflection\ReflectionClass;
-use PHPStan\Node\CollectedDataNode;
-use PHPStan\Type\ObjectType;
-
-final class NetteApplicationUIControl extends AbstractClassTemplateResolver
+final class NetteApplicationUIControl extends AbstractClassMethodTemplateResolver
 {
+    use NetteApplicationUIControlGlobals;
+
     public function getSupportedClasses(): array
     {
         return ['Nette\Application\UI\Control'];
     }
 
-    protected function getClassTemplates(ReflectionClass $reflectionClass, CollectedDataNode $collectedDataNode): array
+    public function getIgnoredClasses(): array
```

Comment:
should be possible to set from config:
example: I have Plugins which are also Controls and I want to resolve them by my own resolver not this built-in

Reply:
Agreed but for now add it as issue and we will figure it out later. Ok?

Reply:
ok